app/controllers/admin/users_controller.rb

```diff
@@ -76,6 +58,11 @@ def set_user
   end
 
   def admin_user
-    @user.admin = true
+    if current_user.admin?
+      return true
+    else
```

Comment:
「管理ユーザーでないならリダイレクトさせる」記載で良いと思います。
```suggestion
    unless currnt_user.admin?
```

Reply:
修正しました。